internal/rest/restgateway.go

```diff
@@ -45,6 +45,8 @@ const (
 // RESTGateway as the HTTP gateway interface for fabconnect
 type RESTGateway struct {
 	config          *conf.RESTGatewayConf
+	processor       tx.TxProcessor
+	receiptStore    receipt.ReceiptStore
 	syncDispatcher  restsync.SyncDispatcher
 	asyncDispatcher restasync.AsyncDispatcher
 	router          *router
```

Comment:
This is outside the scope of this PR, but we may want to re-think having everything roll up to the `RESTGateway` as the top level struct that gets created. `firefly-ethconnect` has this problem where things completely unrelated to REST get tangled up with it, because it's the only point in the code that has access to all other points. So when "child" structs need to pass data or a pointer, the `RESTGateway` ended up needing to do that. Maybe the problem isn't as bad in `firefly-fabconnect`. I'm still exploring the full codebase.